wilsonChan/assignments/slidingwindow/lc209/lc209.py

```diff
@@ -1,8 +1,30 @@
+# Problem Statement #
+
+# Given an array of positive numbers and a positive number ‘S,’ find the length of the smallest contiguous subarray whose sum is greater than or equal to ‘S’. Return 0 if no such subarray exists.
+
```

Comment:
When attempting these problems, you are going to want to write a high-level approach in the event that your interviewer doesn't code in the same language or someone new is trying to review your code.